examples/classification.ipynb

```diff
@@ -5,7 +5,11 @@
    "id": "cb9e5d62",
```

Comment:
<p>Maybe rephrase the second paragraph to make it simpler. Like "It is possible to load ... . But here we show how to do it manually". </p><p> </p><p>Also, should it be <em>from</em> instead of <em>form</em>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Maybe remove the reasoning why we don't use fetch_ucr_dataset. Just note that it is possible to do it with this function, but we load data manually.</p><p>This part may become irrelevant if we consider the possibility of changing the version in the future.</p>